test/core/gpr/alloc_test.cc

```diff
@@ -35,8 +35,8 @@ static void test_custom_allocs() {
   const gpr_allocation_functions default_fns = gpr_get_allocation_functions();
   intptr_t addr_to_free = 0;
   char* i;
-  gpr_allocation_functions fns = {fake_malloc, nullptr, fake_realloc,
-                                  fake_free};
+  gpr_allocation_functions fns = {fake_malloc, nullptr, fake_realloc, fake_free,
+                                  nullptr,     nullptr, nullptr};
```

Comment:
please add a test when these are not nullptr.

Reply:
Done.